### DISCUSSION TITLE
Does this project work for Vendor Central Portal

### DISCUSSION THREAD

**Discussion by peter21101** [saleweaver/python-amazon-sp-api#302] [Feature requests]
I always got below message.
[{'code': 'Unauthorized', 'message': 'Access to requested resource is denied.', 'details': ''}]

I tried to get order data from Vendor Central Portal not Seller Central Portal. Does this project work for Vendor Central Portal? If not, can project owner add features for Vendor Portal?

**Comment by saleweaver** [comment]
Yes it does. "Unauthorized" means you are not authorized to request the resource, though you have to get permissions to, and add them to the app.

**Reply by peter21101** [reply]
Hi Michael,
Thank you for your reply. But I did authorize to request the resources. I tried to authorize all resources using * but it still came out the same results. Is there way(s) to verify the resource authorization for vendor orders through vendor central portal? Thank you very much.
